### DISCUSSION TITLE
Deepgram Redaction (and Audio Redaction)

### DISCUSSION THREAD

**Discussion by alankleindev** [deepgram/community#572] [General help] [answered]
I would like to confirm if Deepgram can redact portions of the actual recording (i.e. bleep noise or white noise) that has sensitive information redacted in the transcript or if there are any suggestions on how to go about that?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by dithyagit** [comment]
I also have the same question. Deepgram Team, please assist with this.

**Comment by jpvajda** [comment]
Just to confirm the question: Your audio has redacted words removed from it already before you pass it to Deepgram? OR do you need Deepgram to Redact the audio ?  You can check out our Docs on our [Redaction ](https://developers.deepgram.com/docs/redaction)feature to learn more about how it works depending on the answer to my question it might work for your needs.

**Comment by davidvonthenen** [comment, marked as answer]
If you are using one of the SDK, there is an option to enable redaction. Python for example:
https://github.com/deepgram/deepgram-python-sdk/blob/main/deepgram/clients/prerecorded/v1/options.py#L105

If it's for live streaming, there is a similar parameter.
https://github.com/deepgram/deepgram-python-sdk/blob/main/deepgram/clients/live/v1/options.py#L75

All the SDKs have something to that effect.
